cmd/nerdctl/network_inspect.go

```diff
@@ -58,50 +61,77 @@ func networkInspectAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
-	netMap, err := e.NetworkMap()
+	mode, err := cmd.Flags().GetString("mode")
+	if err != nil {
+		return err
+	}
+	format, err := cmd.Flags().GetString("format")
 	if err != nil {
 		return err
 	}
 
-	result := make([]interface{}, len(args))
-	for i, name := range args {
-		if name == "host" || name == "none" {
-			return fmt.Errorf("pseudo network %q cannot be inspected", name)
-		}
-		l, ok := netMap[name]
-		if !ok {
-			return fmt.Errorf("no such network: %s", name)
-		}
+	var result []interface{}
+	var errs []string
+	code := 0
+
+	walker := netwalker.NetworkWalker{
+		Client: e,
+		OnFound: func(ctx context.Context, found netwalker.Found) error {
```

Comment:
Probably we need to check if `found.MatchCount == 1` and return error if not. Otherwise a prefix (e.g., `nerdctl network inspect a`) corresponding to multiple ids will print all.

Reply:
done

Reply:
done